src/panels/lovelace/cards/hui-statistics-graph-card.ts

```diff
@@ -73,6 +79,20 @@ export class HuiStatisticsGraphCard extends LitElement implements LovelaceCard {
 
   private _statTypes?: Array<StatisticType>;
 
+  public hassSubscribe() {
+    if (!this._config?.energy_date_selection) {
+      return [];
+    }
```

Comment:
If the config changes, the subscriptions are now not updated, so behaviour in the editor will be wrong.

Reply:
Can you explain more? Are you saying that if a user uses the editor to uncheck `energy_date_selection`, nothing will force the plot to be redrawn with the now-static time period?

Is this moot now that I reverted the editor changes?

Reply:
The subscription is created on init of the element, so if some one enables the `energy_date_selection` option after the element was initialised, it will not add the subscription and thus miss data/not work.

This is not just for the visible editor, but also for the YAML editor.

Reply:
I still think we can not use `hassSubscribe` here, it will not work in the card editor.